src/sentry/models/pullrequest.py

```diff
@@ -106,7 +106,7 @@ class PullRequestComment(Model):
     __relocation_scope__ = RelocationScope.Excluded
 
     external_id = BoundedBigIntegerField()
-    pull_request = FlexibleForeignKey("sentry.PullRequest")
+    pull_request = FlexibleForeignKey("sentry.PullRequest", db_constraint=False)
```

Comment:
This isn't a unique constraint, I don't understand why we need to remove it here?

Reply:
or i might be wrong and the unique_together constraint wasn't added in the first place?

Reply:
okay upon second glance i actually don't need to do this. we are missing the uniqueness constraint with pullrequest + comment_type so I can just add that

Reply:
i updated the PR